packages/libs/deer/README.md

```diff
@@ -0,0 +1,86 @@
+# Deer
```

Comment:
I don't _think_ we want to capitalize this here, as we don't anywhere else (similar to `error-stack`'s README](https://github.com/hashintel/hash/tree/dw/mixed-updates/packages/libs/error-stack#error-stack))

Reply:
fixed in [2bb0b1b](https://github.com/hashintel/hash/pull/1163/commits/2bb0b1b818abf9c9c3622249777410805a730abc)